community-operators/federation/federation.v0.0.5.clusterserviceversion.yaml

```diff
@@ -0,0 +1,189 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: federation.v0.0.5
+  namespace: placeholder
+  annotations:
```

Comment:
Additional Operator Hub annotations can be found here: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md

Reply:
Added the annotations that I know. I don't have a person's name that I'm willing to put for support at this point. PTAL